tests/performance/scripts/runperf.sh

```diff
@@ -49,6 +49,7 @@ if [ $# -eq 0 ]
     #run testcase
```

Comment:
can we add some --help to indicate the different kinds of tests which can be run using this script ?

Reply:
Thanks.